README.md

```diff
@@ -1,13 +1,13 @@
 # Python Tools for Visual Studio
 
-[![Join the chat at https://gitter.im/Microsoft/PTVS](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Microsoft/PTVS?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+PTVS is a free, open source plugin that turns Visual Studio into a Python IDE. 
 
-Welcome to our new home on github!
+[![Join the chat at https://gitter.im/Microsoft/PTVS](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Microsoft/PTVS?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Downloads of [PTVS 2.1](https://pytools.codeplex.com/releases/view/109707) and [PTVS 2.2 RC](https://pytools.codeplex.com/releases/view/614624) are still available from CodePlex, but new releases will be here and linked from [microsoft.github.io/PTVS](https://microsoft.github.io/PTVS).
+[PTVS 2.2 RC2](https://github.com/Microsoft/PTVS/releases/v2.2-rc2) is now available for download from our Releases section. [PTVS 2.1](https://pytools.codeplex.com/releases/view/109707) is still available from CodePlex.
 
-We're still working on the transfer, but we are mostly up-and-running. You may notice a bit of experimentation as we figure out the best way to move our team from our old system.
+Visit our page on [VisualStudio.com](https://www.visualstudio.com/en-us/explore/python-vs) for an overview and videos, and check out our [Getting Started](http://aka.ms/PTVSTutorial) series on YouTube.
 
-Our [documentation](https://github.com/Microsoft/PTVS/wiki) is now on the wiki, and is _mostly_ looking good (feel free to help us out). Also, the [issue tracker](http://github.com/Microsoft/PTVS/issues) here is where we are managing bugs and feature requests, as well as any questions you may have. We are not planning to transfer all our [existing issues](https://pytools.codeplex.com/workitem/list/advanced) from CodePlex, so if you have one there that is really important to you, please recreate it with a link to the original so we don't lose it.
+Our [documentation](https://github.com/Microsoft/PTVS/wiki) is now on the wikigood (feel free to help us improve it), and we manage all of our issues here on the [issue tracker](http://github.com/Microsoft/PTVS/issues) or in the gitter chat linked above. Pull requests and feature suggestions are welcome - see the [Contributing](https://github.com/Microsoft/PTVS/wiki#contributing) section for information.
```

Comment:
Drop the "in gitter chat".  We should track issues in github to ensure they get resolved and to ensure history of answers is easily searchable.